code/datums/callback.dm

```diff
@@ -117,6 +117,10 @@
 	if (!object)
 		return
 
+	if(fexists(object))
+		to_chat(usr, "[object] may be an external library. Calling external libraries is disallowed.", confidential = TRUE)
```

Comment:
this should throw a log

Reply:
I had the exact same thought, but when I asked, Mothblocks said a `to_chat` was good enough.

Reply:
attempted escalation should absolutely be logged, idk what mothblocks is smoking